src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -1885,24 +1886,29 @@ private LineChart GetChartForCompound(string compoundName)
     }
 
     private float GetCompoundAmount(Patch patch, string compoundName)
+    {
+        return GetCompoundAmount(patch, patch.Biome, compoundName);
+    }
+
+    private float GetCompoundAmount(Patch patch, BiomeConditions biome, string compoundName)
```

Comment:
isn't the patch parameter unused here now?

Reply:
never mind, it's used for the chunk amounts. Had to open my editor to see where it was used